src/main/resources/config.yml

```
@@ -152,3 +152,15 @@ errors:
 
     # Trying to message a team, but not in one
     not-in-team: "<red>(!) You are not in a team.</red>"
+
+
+pm-sound:
+    # Enable or disable playing a sound when a private message is received
+    enabled: true
+    # Sound to play on private message (Minecraft sound identifier)
+    # Examples: ENTITY_ALLAY_AMBIENT_WITH_ITEM, ENTITY_PLAYER_LEVELUP, etc.
+    sound: BLOCK_NOTE_BLOCK_PLINGt
+    # Volume of the sound (float). 1.0 is normal volume, increase to be louder.
+    volume: 1.0
+    # Pitch of the sound (float). 1.0 is normal pitch, lower for deeper, higher for sharper.
+    pitch: 1.0
```

Some rewording of the explanations to be more consistent with the rest of the config.

```suggestion
    # Whether to play a sound when receiving a private message
    enabled: false
    # Minecraft sound identifier from https://jd.papermc.io/paper/1.20.4/org/bukkit/Sound.html
    sound: BLOCK_NOTE_BLOCK_PLING
    # Volume of the sound from 0.0 to 1.0. Decrease to be quieter.
    volume: 1.0
    # Pitch of the sound from 0.5 to 2.0. Values below 1.0 lower the pitch and increase the duration; values above 1.0 raise the pitch and reduce the duration.
    pitch: 1.0
```